src/content/docs/style-guide/writing-docs/writer-workflow/github-troubleshooting.mdx

```diff
@@ -9,224 +9,281 @@ redirects:
 
-    1. In the terminal, ensure you're in the docs-website directory.
-    2. Hit CONTROL+C to stop yarn, if necessary.
-    3. Blow away modules and cache and start from scratch by running the `kknuke` Keyboard Maestro command, or run `yarn reboot` in your terminal.
+  `const parse = require('rehype-parse');`
+  
+  3. Save the file and commit the change to your PR.
+  4. Re-run the build checks.
+  5. Wait a LOOOOONG time.
 
-       <Callout variant="important">
-         If you use the `kknuke` Keyboard Maestro command, when everything completes, start the site by running `yarn start`.
-       </Callout>
-  </Collapser>
+## PR merge conflicts [#merge-conflict]
 
-  <Collapser
-    className="freq-link"
-    id="run-incognito"
-    title="Run your local build in private mode"
-  >
-    Sometimes the local site builds, but pages within the site don't. Running the local build in a private/incognito session may to fix this issue. You can also try clearing out your browser's cache.
-  </Collapser>
-</CollapserGroup>
+### Problem
 
-## My redirect throws a 404 error when testing it locally [#redirect-404]
+When you try merge a PR from your local branch, you get a merge conflict.
 
-Redirects are a bit strange on local builds. To test them, navigate to the page that is being redirected, wait until it throws a 404, and then wait ~1-2 minutes. It should redirect you after a while. If it doesn’t, ensure you set up the redirect correctly.
+### Solution
 
-You can also test the redirects out in the Gatsby build.
+If you're getting a merge conflict, click **Resolve conflict** in either Github desktop or browser and review the conflicts line by line. Make sure to get your approver to review your PR after you fix the merge conflict.
 
-## A check fails in the PR [#check-fail]
+#### Avoiding merge conflicts
+Here are some recommendations to avoid merge conflicts:
 
-<Callout variant="important">
-  The only checks needed to merge a PR are the checks marked as `required` on the PR. These are `run linter`, `run tests`, `license/cla`, and `unpaired translations removed` for merges to develop, and `build the docs site` for merges to main. If a required check fails, the failure must be addressed in order for the PR to be merged. If an optional check fails, reach out in the help channel so that the hero can look into the failure, but feel free to merge the PR since optional checks don't block releases.
-</Callout>
+* Before you merge to develop or main, click **Fetch origin** to make sure you pull down the most up-to-date version of develop or main. 
+* If you’re working on taxonomy changes, fix your merge conflict as soon as possible. If your branch lingers for a while, it can get outdated from develop pretty fast and that can cause some unexpected issues.
+* If you get a conflict merge from what's new posts, it's because the `whats-new-ids.json` file (which is automatically updated when the site builds) can get out-of-date pretty fast.
```

Comment:
@bradleycamacho or @rhetoric101 , what are we supposed to do if the `whats-new-ids.json` file is out of date? Fetch origin to make sure it's up to date?

Reply:
Honestly no idea, I've never run into this before! @rhetoric101 probably had a better idea